tests/recipes/noarch_selector_variants/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.34.41" %}
+{% set hash = "3a6943c75a0d292ab6e008bce58ee6503776969479f991f5ad03a5d877af29ae" %}
+
+package:
+  name: botocore
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/b/botocore/botocore-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 1
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  build:
+    - cross-python_{{ target_platform }}    # [build_platform != target_platform]
+    - python                                # [build_platform != target_platform]
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
@0xbe7a Can you fix this recipe? I doesn't matter for the test but it should have a python constraint based on `python_leq_310`.